### DISCUSSION TITLE
Allow static content consumed in the Model Card

### DISCUSSION THREAD

**Discussion by hanneshapke** [tensorflow/model-card-toolkit#229] [Ideas]
Non-ML experts should be able to contribute to Model Cards. Therefore, model cards should support static content from other sources than the initial source code, e.g. a text file on Google Cloud Storage, S3, etc., and it should be included in the fully rendered model card.

**Comment by codesue** [comment]
Is this related to https://github.com/tensorflow/model-card-toolkit/issues/109?
